docs/3x/PT_WeightOnlyQuant.md

```diff
@@ -0,0 +1,268 @@
+
+
+#### RTN
+|  rtn_args  | comments |                                 default value                            |
+|----------|-------------|-------------------------------------------------------------------|
+|               group_dim (int)       |  Dimension for grouping                                 |  Default is 1      |
+|               use_full_range (bool) |  Enables full range for activations                     |  Default is False  |
+|               use_mse_search (bool) |  Enables mean squared error (MSE)   search              |  Default is False  |
+|               use_layer_wise (bool) |  Enables quantize model per layer                       |  Defaults to False |
+|               model_path (str)      |  Model path that is used to load   state_dict per layer |                    |
+
+``` python
+# Quantization code
+from neural_compressor.torch.quantization import prepare, convert, RTNConfig
+
+quant_config = RTNConfig()
+model = prepare(model, quant_config)
+model = convert(model)
+```
+
+#### GPTQ
+|  gptq_args  | comments |      default value                                                       |
+|----------|-------------|-------------------------------------------------------------------|
+|               use_mse_search (bool)   |  Enables mean squared error (MSE) search                                                                                                   |  Default is False 
+|               use_layer_wise (bool)   |  Enables quantize model per layer                                                                                                          |  Defaults to False |
+|               model_path (str)        |  Model path that is used to load   state_dict per layer                                                                                    |                    |
+|               use_double_quant (bool) |  Enables double quantization                                                                                                               |  Default is False  |
+|               act_order (bool)        |  Whether to sort Hessian's diagonal   values to rearrange channel-wise quantization order                                                  |  Default is False  |
+|               percdamp (float)        |  Percentage of Hessian's diagonal   values' average, which will be added to Hessian's diagonal to increase   numerical stability           |  Default is 0.01.  |
+|               block_size (int)        |  Execute GPTQ quantization per   block, block shape = [C_out, block_size]                                                                  |  Default is 128     |
+|               static_groups (bool)    |  Whether to calculate group wise   quantization parameters in advance. This option mitigate actorder's extra   computational requirements. |  Default is False.  |
```

Comment:
Please remove `Default is`, it's already shown in the column title.

Reply:
Thanks, removed https://github.com/intel/neural-compressor/pull/1824/commits/86f99577899704cbf0d614502cd7b6f5b21ca17e